etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -113,6 +115,7 @@ var (
 	ErrPermissionDenied     = Error(ErrGRPCPermissionDenied)
 	ErrRoleNotGranted       = Error(ErrGRPCRoleNotGranted)
 	ErrPermissionNotGranted = Error(ErrGRPCPermissionNotGranted)
+	ErrRetryAuthRequired    = Error(ErrGRPCRetryAuthRequired)
```

Comment:
I don't think this should be something ever returned by the server; if a retry is necessary the server can retry internally